src/luarocks/fetch/cvs.lua

```diff
@@ -20,9 +20,14 @@ function cvs.get_sources(rockspec, extract, dest_dir)
    assert(type(rockspec) == "table")
    assert(type(dest_dir) == "string" or not dest_dir)
 
+   local cvs_cmd = rockspec.variables.CVS
+   if not fs.execute_quiet(cvs_cmd, "--version") then
+      return nil, "'"..cvs_cmd.."' program not found. Is CVS installed? You may want to edit variables.CVS"
```

Comment:
alternatively, `"'"..cvs_cmd.."' program not found. Please make sure it is available in your PATH".` _might_ be an easier instruction to follow (not sure).